mongo/integration/crud_helpers_test.go

```diff
@@ -56,6 +56,20 @@ func createUpdate(mt *mtest.T, updateVal bson.RawValue) interface{} {
 	return nil
 }
 
+// create a hint string or document from a bson.RawValue
+func createHint(mt *mtest.T, val bson.RawValue) interface{} {
```

Comment:
Add `mt.Helper` as the first line of this function so testing errors will show the function's caller if we hit the `Fatalf` in the `default` case.

Reply:
done